x/ibc/20-transfer/keeper/callbacks.go

```diff
@@ -6,36 +6,36 @@ import (
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
-	channeltypes "github.com/cosmos/cosmos-sdk/x/ibc/04-channel/types"
-	porttypes "github.com/cosmos/cosmos-sdk/x/ibc/05-port/types"
+	channel "github.com/cosmos/cosmos-sdk/x/ibc/04-channel"
+	channelexported "github.com/cosmos/cosmos-sdk/x/ibc/04-channel/exported"
+	port "github.com/cosmos/cosmos-sdk/x/ibc/05-port"
 	"github.com/cosmos/cosmos-sdk/x/ibc/20-transfer/types"
 	ibctypes "github.com/cosmos/cosmos-sdk/x/ibc/types"
 )
 
 // nolint: unused
-func (k Keeper) onChanOpenInit(
+func (k Keeper) OnChanOpenInit(
 	ctx sdk.Context,
-	order channeltypes.Order,
+	order channel.Order,
 	connectionHops []string,
 	portID,
 	channelID string,
-	counterparty channeltypes.Counterparty,
+	counterparty channel.Counterparty,
 	version string,
 ) error {
-	if order != channeltypes.UNORDERED {
-		return channeltypes.ErrInvalidChannel(k.codespace, "channel must be UNORDERED")
+	if order != channel.UNORDERED {
+		return channel.ErrInvalidChannel(k.codespace, "channel must be UNORDERED")
 	}
 
 	// NOTE: here the capability key name defines the port ID of the counterparty
 	if counterparty.PortID != k.boundedCapability.Name() {
-		return porttypes.ErrInvalidPort(
+		return port.ErrInvalidPort(
 			k.codespace,
-			fmt.Sprintf("counterparty port ID doesn't match the capability key (%s ≠ %s)", portID, k.boundedCapability.Name()),
-		)
+			fmt.Sprintf("counterparty port ID doesn't match the capability key (%s ≠ %s)", counterparty.PortID, k.boundedCapability.Name()))
 	}
 
 	if strings.TrimSpace(version) != "" {
-		return ibctypes.ErrInvalidVersion(k.codespace, "version cannot be blank")
+		return ibctypes.ErrInvalidVersion(k.codespace, "version must be blank")
```

Comment:
```suggestion
		return ibctypes.ErrInvalidVersion(k.codespace, "version cannot be blank")
```

Reply:
In the ICS20 spec, the version string is empty.
see: https://github.com/cosmos/ics/blob/master/spec/ics-020-fungible-token-transfer/README.md#routing-module-callbacks